be-python-flask/files/tests/requirements.txt

```diff
@@ -0,0 +1,5 @@
+flask==1.1.1
+mypy==0.760
+flake8==3.7.9
+pytest==5.3.2
+pytest-cov==2.8.1
```

Comment:
pytest-cov 2.8.1 requires coverage>=4.4. When using coverage 5.x (in another context) I ran into an issue as the base image python did not compile the sqlite module (https://github.com/nedbat/coveragepy/issues/887)
Perhaps we should specify coverage==5.0.3 here and make sure we have a  python supporting sqlite or pin an earlier version.